ci/delete_thin-edge/delete_thin_edge.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
Doesn't it work with `sh`? If yes, change it.
```suggestion
#!/bin/sh
```

Reply:
fixed